tests/linearizability/linearizability_test.go

```diff
@@ -94,30 +78,31 @@ func testLinearizability(ctx context.Context, t *testing.T, config e2e.EtcdProce
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer clus.Close()
+	defer clus.Stop()
 	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+
 	go func() {
-		defer cancel()
 		err := triggerFailpoints(ctx, t, clus, failpoint)
 		if err != nil {
 			t.Error(err)
 		}
 	}()
 	operations := simulateTraffic(ctx, t, clus, traffic)
-	clus.Close()
+	time.Sleep(10 * time.Second)
```

Comment:
I would not recommend removing cluster.Stop here as this is needed to saftly copy out the data dir.